data/human/hails.txt

```diff
@@ -2636,17 +2647,17 @@ phrase "friendly navy"
 		"navy purpose b" 4
 		"navy purpose c"
 	word
-		". "
+		"."
 	word
 		"" 5
-		"Your cooperation is greatly valued."
-		"We thank you for your cooperation."
-		"We appreciate your complicance."
-		"Be thankful for our protection."
-		"Be grateful for our protection."
-		"Be thankful for our sacrifice."
-		"Be grateful for our sacrifice."
-		"Remember that our sacrifice is for your sake."
+		" Your cooperation is greatly valued."
+		" We thank you for your cooperation."
+		" We appreciate your complicance."
+		" Be thankful for our protection."
+		" Be grateful for our protection."
+		" Be thankful for our sacrifice."
+		" Be grateful for our sacrifice."
+		" Remember that our sacrifice is for your sake."
```

Comment:
Why the space to start instead of leaving as-is? This seems to go against most of your existing changes in this PR.

Reply:
It's because of the `"" 5`. If the space is in a separate thing, then there is the chance that there will be a space at the end of a hail. I know that nobody sees that, but I don't want extra spaces.

If you think that it would be better, or make more sense for my PR, then I'll revert it.

Reply:
Is your approval review and thumbs up emoji saying that this change is fine?

Reply:
Sorry! Had a couple things come up that distracted me for a little while. Merging!